appveyor.yml

```diff
@@ -0,0 +1,51 @@
+environment:
+  # There is no need to run the build for all the Python version /
+  # architectures combo as the generated nilearn wheel is the same on all
+  # platforms (universal wheel).
+  # We run the tests on 2 different target platforms for testing purpose only.
+  # We use miniconda versions of Python provided by appveyor windows images
+  matrix:
```

Comment:
We should adapt the configuration to the same configuration than travis with the same supported version.
Also it should be a discussion between 32/64 bits support